### PR TITLE
changed the nicfi landing page picture

### PR DESCRIPTION
**Pull request by teopista**

### CODE REVIEW

**Comment by 12rambau** [comment]
hi teopista is this PR ready for review (it's on stale since last week)?

**Review by 12rambau — CHANGES_REQUESTED** [review]
Thanks for contributing to the documentation. Inputs from someone that is using the tool on daily basis is extremely valuable to us ! Tere are few changes that I would like you to make before merging: 
- the name of the image is including an useless number could you make it "signup.png" instead of "signup"
- I think you need to change the GIF or remove it entirely as it embeds many flows: 
  - it exposes you personnal phone, name, surname email. In short everything I need to hack you on the web. 
  - it is badly cut, I see the chrome popup window on the right when you validated the form.

File: docs/source/setup/nicfi.rst [hunk 1]
```diff
@@ -33,12 +33,29 @@ The first step is to make sure you have a GEE account. You can follow `this tuto
 Sign up for NICFI imagery
 -------------------------
 
-Sign up for NICFI Planet data access using the same email address associated with your GEE account. To sign up for access to the NICFI data program, go to `<www.planet.com/nicfi/>`_. In the field, **What is your GEE identity?**, enter the email address that has access to GEE.
+Sign up for NICFI Planet data access using the same email address associated with your GEE account. To sign up for access to the NICFI data program, go to `<https://www.planet.com/nicfi/>`_.
```
Comment (12rambau) [inline docs/source/setup/nicfi.rst:36]:
```suggestion
Sign up for NICFI Planet data access using the same email address associated with your GEE account. To sign up for access to the NICFI data program, go to `<https://www.planet.com/nicfi/>`_.
```

File: docs/source/setup/nicfi.rst [hunk 2]
```diff
@@ -33,12 +33,29 @@ The first step is to make sure you have a GEE account. You can follow `this tuto
...
-Sign up for NICFI Planet data access using the same email address associated with your GEE account. To sign up for access to the NICFI data program, go to `<www.planet.com/nicfi/>`_. In the field, **What is your GEE identity?**, enter the email address that has access to GEE.
+Sign up for NICFI Planet data access using the same email address associated with your GEE account. To sign up for access to the NICFI data program, go to `<https://www.planet.com/nicfi/>`_. 
 
-.. thumbnail:: ../_images/setup/nicfi/landing.png
+.. thumbnail:: ../_images/setup/nicfi/nicfi_page.png
    :title: Planet NICFI landing page.
    :group: setup_nicfi
 
+
+Click on the **Sign up** button and fill in the form.
```
Comment (12rambau) [inline docs/source/setup/nicfi.rst:43]:
```suggestion
Click on the **Sign up** button and fill in the form.
```

File: docs/source/setup/nicfi.rst [hunk 3]
```diff
@@ -33,12 +33,29 @@ The first step is to make sure you have a GEE account. You can follow `this tuto
...
 
+
+Click on the **Sign up** button and fill in the form. 
+
+.. thumbnail:: ../_images/setup/nicfi/signup.png
+   :title: Sign up form.
+   :group: setup_nicfi
+
+
+After submitting the form, an email will be sent with a link to activate your account.
```
Comment (12rambau) [inline docs/source/setup/nicfi.rst:50]:
```suggestion
After submitting the form, an email will be sent with a link to activate your account.
```

**Review by 12rambau — APPROVED** [review]
Nice job, thanks for your contribution!

**Comment by teopista** [comment]
> 

You're welcome! Thank you too.

**Pull request merged**